### ISSUE TITLE
Modulo operator is not supported

### ISSUE THREAD

**Issue by VladimirPlokhov** [sasstools/sass-lint#565]
Using modulo operator `%` in the Scss code:

```scss
$my-variable: 5 % 2;
```

causes this error:

```
error Please check validity of the block starting from line #1  Fatal
```

**Comment by DanPurdy** [comment]
Hi @VladimirPlokhov This is a parse issue with Gonzales-pe our AST. It's fixed in their latest release though which we will be moving up to in 1.6 which is just waiting on them to fix some windows installation issues.

**Comment by DanPurdy** [comment]
@VladimirPlokhov So this is working for `.scss` format in sass-lint 1.6 but gonzales still throws a parse error in `.sass` format, we'll look to fix that soon. I'll create a new issue specifically for `.sass` format.

#636
https://github.com/DanPurdy/gonzales-pe/issues/6
https://github.com/tonyganch/gonzales-pe/issues/165
